rocket.py

```diff
@@ -0,0 +1,91 @@
+import math
+
+import numpy as np
+from rich import print
+
+from functions import *
+from pid import PID
+
+
+GRAVITY = 9.8
+
+
+class Rocket:
+	def __init__(self, dry_mass, fuel, altitude, velocity, fuel_consumption, thrust):
+		# Massive properties
+		self.dry_mass = dry_mass
+		self.fuel = fuel
+
+		# Location & Motion
+		self.altitude = altitude
+		self.velocity = velocity
+		self.acceleration = 0
+
+		# Operational & Engine
+		self.fuel_consumption = fuel_consumption
+		self.thrust = thrust
+		self.throttle = 0
+		
+	@property
+	def total_mass(self):
+		return self.dry_mass + self.fuel
+
+	def dict(self):
+		return {
+			'dry_mass': self.dry_mass,
+			'fuel': self.fuel,
+
+			'altitude': self.altitude,
+			'velocity': self.velocity,
+			'acceleration': self.acceleration,
+
+			'fuel_consumption': self.fuel_consumption,
+			'thrust': self.thrust,
+			'throttle': self.throttle,
+
+			'estimated_distance': self.estimated_distance()
+		}
+
+	def reset_motion(self):
+		self.altitude = 1
+		self.velocity = 0
+		self.acceleration = 0
```

Comment:
Why do we need a function for this? It's only used once, and It's not even used properly.